tests/business-logic/User.test.ts

```diff
@@ -0,0 +1,84 @@
+import UserEntity from "@business-logic/User";
+
+import { hashPassword } from "@helpers/auth";
+import prisma from "@helpers/prisma";
+import { teardown } from "@helpers/tests/teardown";
+
+let hashedPassword: string;
+const email = "doe@email.com";
+
+describe("User", () => {
+  beforeEach(async () => {
+    await teardown();
+    hashedPassword = await hashPassword("Doe@2022^3");
+  });
+
+  describe("#create", () => {
+    it("should create user", async () => {
+      const requestParams = {
+        name: "John Doe",
+        email,
+        password: hashedPassword,
+      };
+      const entity = new UserEntity();
+      const result = await entity.create(requestParams);
+      const { message } = result;
+      expect(message).toBe("User created");
+    });
+    it("Should not create user when email exists", async () => {
+      const requestParams = {
+        name: "John Doe",
+        email,
+        password: hashedPassword,
+      };
+      const entity = new UserEntity();
+      await entity.create(requestParams);
+      await expect(async () => {
+        await entity.create(requestParams);
+      }).rejects.toThrowError("Email address is already registered.");
+    });
+    it("Should not create user when password is invalid", async () => {
```

Comment:
just a small thing and personal preference. can you add an empty paragraph between the it('...') blocks.

I also prefer paragraphs inside the blocks

```javascript
  it("should return an existing user", async () => {
    const company = await prisma.company.create({
      data: {
        name: "Academy Records",
      },
    });

    const user = await prisma.user.create({
      data: {
        name: "Chris Rock",
        email: "test2@nueno.com",
        password: passwordHash,
        companyId: company.id,
      },
    });

    const userEntity = new UserEntity();
    const result = await userEntity.find(user.id);

    expect(result?.name).toBe(user.name);
  }, 4000);
```

Reply:
I have prettier as my default file formatter

Reply:
interesting, prettier doesn't format that for me and the settings should be the same